src/BigBlueButton.php

```diff
@@ -61,21 +64,22 @@ class BigBlueButton
     protected $jSessionId;
 
     /**
-     * @param  string     $baseUrl (optional)
-     * @param  string     $secret  (optional)
+     * @param string $baseUrl (optional)
```

Comment:
In my opinion, a pr should be as self contained as possible. So those style changes are polluting the whole pr and I guess they will break with cs-fixer. Would be nice if we could merge my pr to run cs-fixer also on travis.